doc/modules/manifold.rst

```diff
@@ -319,7 +319,7 @@ function :func:`spectral_embedding` or its object-oriented counterpart
 Complexity
 ----------
 
-The Spectral Embedding algorithm comprises three stages:
+The Laplacian Eigenmaps algorithm comprises three stages:
```

Comment:
I would prefer "The Spectral Embedding (Laplacian Eigenmaps) algorithm..."

Spectral embedding is an algorithm commonly used to denote this problem (see for instance
http://www.cs.cmu.edu/~aarti/Class/10701/readings/Luxburg06_TR.pdf
that has 4000 citations)